### PR TITLE
Summon new features 24.06.00

### PR DESCRIPTION
**Pull request by AlexanderBlanchardAC**
This patch adds several new features to Summon, including the ability to control whether the Explore More Section is visible and to control whether available book covers are displayed when using the Summon Searcher.

Testing Book Covers:
This patch adds the ability to use only book covers generated by Aspen or to include those available from the API when carrying out a Summon Search. Choosing to use generated covers only increases the speed at which Summon loads results.

TEST PLAN:
(After applying patch).

1. Carry out a Summon Search as usual. You will notice that you are returned book covers that are generated by Aspen.
2. Navigate to Primary Configurations -> Library Systems and then edit whichever Library is your default.
3. Scroll to the Summon Section and check the tick box that is labelled, 'Show Available Covers in Summon.'
4. Click to save your changes and repeat your search.
5. Notice that the covers you are returned are now a mix of generated covers and actual book covers from the API - note that your results may load more slowly with this option enabled.

Testing Explore More Section:
This patch adds the ability to control whether the Explore More Section is visible in search results. By default, it is set to visible.

TEST PLAN

1. Add the patch
2. Carry out a a Summon Search
3. Note that part way down your results list you will have a box labelled 'Explore More.'
4. Navigate to the Layout Settings: Aspen Administration -> Theme and Layout -> Layout Settings.
5. Within the layout settings, scroll to the checkbox labelled 'Show Explore More Options' and untick it.
6. Repeat your search and notice that this time you do not have the 'Explore More' box in your results.

### CODE REVIEW

File: code/folio_export/folio_export.jar
Comment (mdnoble73) [inline code/folio_export/folio_export.jar:1]: Can we remove this from the PR?

File: code/web/interface/themes/responsive/images/summon.png:Zone.Identifier
Comment (mdnoble73) [inline code/web/interface/themes/responsive/images/summon.png:Zone.Identifier:1]: What is this used for?

File: code/web/release_notes/24.06.00.MD [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Aspen Discovery Updates
 //mark
+
+
+- Added option for Explore More bar to be hidden (*AB*)
```
Comment (mdnoble73) [inline code/web/release_notes/24.06.00.MD]: Can you add a comment with your name(s) ahead of these changes just to make merges easier during lots of updates

File: code/web/sys/DBMaintenance/version_updates/23.12.01.php
Comment (mdnoble73) [inline code/web/sys/DBMaintenance/version_updates/23.12.01.php:1]: Let's remove this change

File: code/web/sys/DBMaintenance/version_updates/24.05.00.php
Comment (mdnoble73) [inline code/web/sys/DBMaintenance/version_updates/24.05.00.php:1]: Let's also remove this change

File: code/web/sys/Theming/LayoutSetting.php
Comment (mdnoble73) [inline code/web/sys/Theming/LayoutSetting.php:1]: Explore more is a key feature of Aspen if we want to disable Explore more, I think we'll want more control than just on and off everywhere.  Can you explain more what your partner wants?

**Review by mdnoble73 — CHANGES_REQUESTED** [review]
See comments in individual files

File: code/web/interface/themes/responsive/images/summon.png:Zone.Identifier
Comment (AlexanderBlanchardAC) [reply code/web/interface/themes/responsive/images/summon.png:Zone.Identifier:1]: This image is used for the Summon option in the Explore More section.

Comment (mdnoble73) [reply code/web/interface/themes/responsive/images/summon.png:Zone.Identifier:1]: I was wondering about the file with the Zone.Identifier at the end, I haven't seen that type of file before

**Pull request closed without merging**